examples/Halo model.ipynb

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
Comments about the notebook:
1. Should use only numpy math functions instead of doing `import math`.
1. Should add units too the plots.
Otherwise, looks good!

Reply:
Done